blacklist/blacklist/Common/Utils/StyleSheet.swift

```diff
@@ -29,5 +29,10 @@ struct StyleSheet {
             static let emptyHeaderText = UIColor(hex: 0x222831)
             static let emptySubHeaderText = UIColor(hex: 0x222831)
         }
+
+        struct Debtors {
+            static let background = #colorLiteral(red: 0.864806354, green: 0.8960095048, blue: 0.8077570796, alpha: 1)
```

Comment:
Try to avoid literal colors, is great in Xcode but we have to try to avoid macros. Use the hexadecimal representation instead. `UIColor(hex: 0xFF00FF)`